src/main/java/com/rarchives/ripme/ripper/rippers/ImagebamRipper.java

```diff
@@ -143,6 +143,7 @@ private void fetchImage() {
                         .get();
 
                 // Find image
+                @SuppressWarnings("unused")
                 Elements metaTags = doc.getElementsByTag("meta");
```

Comment:
Don't suppress warnings if possible. Figure out what was intended and update the code accordingly. In this case it might simply be that the variable is unused and was left behind by a past change, and should now be removed. That looks to be the case here.

Reply:
Again, please don't suppress these warnings. Fix them.